packages/lib-grommet-theme/src/index.js

```diff
@@ -47,7 +48,8 @@ const neutralColors = [
   darkGreen, // neutral-1
   darkTeal, // neutral-2
   navy, // neutral-3
-  gold // neutral-4
+  gold, // neutral-4
+  blue // neutral-5
```

Comment:
Could you update the changelog for the unreleased section as well as the table list of colors in the readme?

Reply:
Done